tools/Az/Az.psd1

```diff
@@ -122,7 +122,7 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.15.0'; },
                @{ModuleName = 'Az.RedisEnterpriseCache'; RequiredVersion = '1.2.0'; }, 
                @{ModuleName = 'Az.Relay'; RequiredVersion = '2.0.0'; }, 
                @{ModuleName = 'Az.ResourceMover'; RequiredVersion = '1.2.0'; }, 
-               @{ModuleName = 'Az.Resources'; RequiredVersion = '6.13.0'; }, 
```

Comment:
Please sync with main branch to dismiss these changes. Your PR should only contains the `src/CodeSigning`

Reply:
This is done.